wire-ios-data-model/Tests/MLS/MLSServiceTests.swift

```diff
@@ -984,6 +1009,7 @@ class MLSServiceTests: ZMConversationTestsBase, MLSServiceDelegate {
 
     private func test_PerformPendingJoinsRecovery(_ recovery: MLSActionExecutor.ExternalCommitErrorRecovery) {
```

Comment:
just to my knowledge, the fact that this is private is enough in order that Xcode does not run it as a test on itself?

Reply:
Yes